railties/test/generators/app_generator_test.rb

```diff
@@ -623,6 +625,19 @@ def test_inclusion_of_a_debugger
     end
   end
 
+  def test_inclusion_of_rubocop
+    run_generator
+    assert_gem "rubocop-rails-omakase"
+  end
+
+  def test_rubocop_is_skipped_if_required
+    run_generator [destination_root, "--skip-rubocop"]
+
+    assert_no_gem "rubocop"
```

Comment:
Could it be `rubocop-rails-omakase`, instead of `rubocop`?

```diff
-assert_no_gem "rubocop"
+assert_no_gem "rubocop-rails-omakase"
```

Reply:
@koic You're right. Raising a PR.